content/en/docs/Reference/catalog-templates.md

```diff
@@ -235,6 +235,8 @@ This alpha version of the `semver template` has the following goals:
 - minor-version (Y-stream) and major-version (X-stream) versioning capabilities
 - clear mapping between input schema and output FBC attributes
 
+Note: `csv.spec.version` is optional in the API definition, however the field is required if you want to use the `semver template` for your catalog.
```

Comment:
In this context, I think you are burying the lead a little bit.

```suggestion
Note: The semver template depends on the optional `csv.spec.version` field. If you want to use the semver catalog template, you must specify a version in your extension's CSV.
```
Or something.

Reply:
Perhaps the second sentence is a bit redundant upstream, but for things a user "must do" I try to lean in so there is no ambiguity.